backend/tracim_backend/command/cleanup.py

```diff
@@ -355,6 +383,248 @@ def _delete_user_database_info(
         return DeleteResultIds(deleted_user_id, deleted_workspace_ids)
 
 
+class DeleteContentRevisionCommand(AppContextCommand):
+    def get_description(self) -> str:
+        return """delete content_revision from database"""
+
+    def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
+        parser = super().get_parser(prog_name)
+        parser.add_argument(
+            "--dry-run",
+            help="dry-run mode",
+            dest="dry_run_mode",
+            default=False,
+            action="store_true",
+        )
+        parser.add_argument(
+            "-z",
+            "--content-revision-id",
+            nargs="+",
+            help="content revision ids",
+            dest="revision_ids",
+            required=True,
+        )
+        return parser
+
+    def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvironment) -> None:
+        self._session = app_context["request"].dbsession
+        self._app_config = app_context["registry"].settings["CFG"]
+        if self._session.bind.dialect.name == "sqlite":
+            listen(Engine, "connect", set_sqlite_pragma_on)
```

Comment:
An explanation could be appreciated. But, if author is not here, we keep as it :)